issue_processing/jira_issue_processing.py

```diff
@@ -178,30 +178,56 @@ def parse_xml(issue_data, persons):
                 issue['resolveDate'] = d.strftime('%Y-%m-%d %H:%M:%S')
 
+            title = issue_x.getElementsByTagName('title')[0]
+            issue['title'] = title.firstChild.data
+
             link = issue_x.getElementsByTagName('link')[0]
             issue['url'] = link.firstChild.data
 
             type = issue_x.getElementsByTagName('type')[0]
             issue['type'] = type.firstChild.data
+            # new consistent format with GitHub issues. Not supported by the network library yet
+            issue['type_new'] = ['issue', str(type.firstChild.data.lower())]
 
             status = issue_x.getElementsByTagName('status')[0]
             issue['state'] = status.firstChild.data
+            # new consistent format with GitHub issues. Not supported by the network library yet
+            issue['state_new'] = status.firstChild.data.lower()
 
             project = issue_x.getElementsByTagName('project')[0]
             issue['projectId'] = project.attributes['id'].value
 
             resolution = issue_x.getElementsByTagName('resolution')[0]
             issue['resolution'] = resolution.firstChild.data
+            # new consistent format with GitHub issues. Not supported by the network library yet
+            issue['resolution_new'] = [str(resolution.firstChild.data.lower())]
+
+            # consistency to default GitHub labels
+            if issue['resolution'] == "Won't Fix":
+                issue['resolution_new'] = ['wontfix']
+
+            # consistency to default GitHub labels
+            if issue['resolution'] == "Won't Do":
+                issue['resolution_new'] = ['wontdo']
 
             for component in issue_x.getElementsByTagName('component'):
-                components.append(component.firstChild.data)
+                components.append(str(component.firstChild.data))
             issue['components'] = components
 
-            for ref in issue_x.getElementsByTagName('issuelinktype'):
-                refId = ref.getElementsByTagName('issuekey')[0].firstChild.data
-                refType = ref.getElementsByTagName('name')[0].firstChild.data
-                refs.append((refId, refType))
-            issue['references'] = refs
+            # if links are not loaded via api, they are added as a history event with less information
+            if skip_history:
+                issue['history'] = []
+                for ref in issue_x.getElementsByTagName('issuelinktype'):
+                    history = dict()
+                    history['event'] = 'add_link'
+                    history["author"] = dict()
+                    history["author"]["name"] = ''
+                    history["author"]["email"] = ''
+                    history['date'] = ''
+                    history['event_info_1'] = ref.getElementsByTagName('issuekey')[0].firstChild.data
+                    history['event_info_2'] = "issue"
+
+                    issue['history'].append(history)
```

Comment:
Please, use consistent quotation marks.

Reply:
@clhunsen, what do you think: Which quotations marks should be use to be consistent?

Reply:
The problem with the not consistent quotation marks is in the whole file, not only the parts of the file that I worked on. Shall I also change the quotation marks in the other methods?

Reply:
Yes, please. A quick search-replace call will probably suffice.